low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -339,12 +356,60 @@ class SSPLDevDeploy:
         Conf.save(sspl_config_index)
         Conf.save(global_config_index)
 
+    def setup_message_broker(self):
+        # Copy message bus conf file
+        os.makedirs("/etc/cortx/", exist_ok=True)
+        shutil.copyfile(f"{SSPL_BASE_DIR}/low-level/files/opt/seagate/sspl"
+                        "/setup/standalone/message_bus.conf",
+                        "/etc/cortx/message_bus.conf")
+        self.setup_kafka()
+
+    def setup_kafka(self):
+        # Install Kafka dependencies
+        self._execute_cmd("yum install -y java-1.8.0-openjdk-headless")
+
+        file_path = ("https://raw.githubusercontent.com/Seagate/cortx-monitor/"
+                     "main/low-level/files/opt/seagate/sspl/setup/standalone")
+        kafka_version = '2.13-2.7.0'
+        r = urllib.request.urlopen(
+            "http://cortx-storage.colo.seagate.com/releases/cortx/github"
```

Comment:
all these URLs better to maintain at the top for better maintainability later

Reply:
Done